src/main/java/org/assertj/jodatime/api/LocalDateAssert.java

```diff
@@ -0,0 +1,407 @@
+   * @return this assertion object.
+   * @throws AssertionError if the actual {@code LocalDate} is {@code null}.
+   * @throws IllegalArgumentException if other {@code LocalDate} is {@code null}.
+   * @throws AssertionError if the actual {@code LocalDate} is not after or equals to the given one.
+   */
+  public LocalDateAssert isAfterOrEqualTo(LocalDate other) {
+    isNotNull();
+    assertLocalDateParameterIsNotNull(other);
+    if (actual.isBefore(other)) {
+      throw Failures.instance().failure(info, shouldBeAfterOrEqualsTo(actual, other));
+    }
+    return this;
+  }
+
+  /**
+   * Same assertion as {@link #isAfterOrEqualTo(LocalDate)} but the {@link LocalDate} is built from given
+   * String, which must follow ISO8601 format (yyyy-MM-dd) to allow calling {@link LocalDate#LocalDate(Object) LocalDate(Object)}
+   * constructor.
+   * <p>
+   * Example :
+   * <pre><code class='java'> // use String in comparison to avoid conversion
+   * assertThat(new LocalDate(&quot;2000-01-01&quot;)).isAfterOrEqualTo(&quot;2000-01-01&quot;)
+   *                                            .isAfterOrEqualTo(&quot;1999-12-31&quot;);</code></pre>
+   *
+   * @param LocalDateAsString String representing a {@link LocalDate}.
+   * @return this assertion object.
+   * @throws AssertionError if the actual {@code LocalDate} is {@code null}.
+   * @throws IllegalArgumentException if given String is null or can't be converted to a {@link LocalDate}.
+   * @throws AssertionError if the actual {@code LocalDate} is not after or equals to the {@link LocalDate}
+   *           built from given String.
+   */
+  public LocalDateAssert isAfterOrEqualTo(String LocalDateAsString) {
+    assertLocalDateAsStringParameterIsNotNull(LocalDateAsString);
+    return isAfterOrEqualTo(new LocalDate(LocalDateAsString));
+  }
+
+  /**
+   * Verifies that the actual {@code LocalDate} is <b>strictly</b> after the given one.
+   * <p>
+   * Example :
+   * <pre><code class='java'> assertThat(new LocalDate(&quot;2000-01-01&quot;)).isAfter(new LocalDate(&quot;1999-12-31&quot;));</code></pre>
+   *
+   * @param other the given {@link LocalDate}.
+   * @return this assertion object.
+   * @throws AssertionError if the actual {@code LocalDate} is {@code null}.
+   * @throws IllegalArgumentException if other {@code LocalDate} is {@code null}.
+   * @throws AssertionError if the actual {@code LocalDate} is not strictly after the given one.
+   */
+  public LocalDateAssert isAfter(LocalDate other) {
+    Objects.instance().assertNotNull(info, actual);
+    assertLocalDateParameterIsNotNull(other);
+    if (!actual.isAfter(other)) {
+      throw Failures.instance().failure(info, shouldBeAfter(actual, other));
+    }
+    return this;
+  }
+
+  /**
+   * Same assertion as {@link #isAfter(LocalDate)} but the {@link LocalDate} is built from given a String that
+   * must follow ISO8601 format (yyyy-MM-dd)to allow calling {@link LocalDate#LocalDate(Object) LocalDate(Object)}
```

Comment:
missing space `(yyyy-MM-dd)to` -> `(yyyy-MM-dd) to`

Reply:
Fixed